src/Microsoft.Extensions.Localization/ResourceManagerStringLocalizerFactory.cs

```diff
@@ -50,6 +50,29 @@ public class ResourceManagerStringLocalizerFactory : IStringLocalizerFactory
             }
         }
 
+        protected virtual string GetKey(TypeInfo typeInfo)
```

Comment:
Can we check if "key" is the appropriate noun here? This is about returning the name of the embedded resource, yes? Let's check what the framework APIs refer to this as.